test/widgets/test_moc.py

```diff
@@ -25,7 +25,7 @@
 import pytest
 
 import libqtile.config
-from libqtile.widget import moc
+from libqtile.widget import base, moc
```

Comment:
There is a unused import here which cause `ruff` to fail, see: https://github.com/qtile/qtile/pull/5420/commits/b1fa16710b49d8f6e018cd8ec0bd5f70debc00c9

Reply:
It's not any more, it was a bug in my fix for the tests. It should be used now, so I think you can drop that patch from your branch.